<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acceuil</class>
 <widget class="QMainWindow" name="acceuil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="prenom_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ex : Albert</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inscription </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Pr√©nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nom_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ex : Olivier</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ex : xyz@gmail.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse e-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>20</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>33</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>34</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>36</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>37</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>39</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>41</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>42</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>43</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>44</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>45</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>46</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>47</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>48</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>49</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>51</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>52</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>53</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>54</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>55</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>58</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>59</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>61</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>62</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>63</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>65</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>66</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>67</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>68</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>69</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>70</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>71</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>72</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>73</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>74</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>76</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>77</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>78</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>79</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>81</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>82</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>83</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>84</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>85</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>86</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>87</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>88</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>89</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>91</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>92</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>93</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>94</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>95</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>97</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>98</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>99</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>101</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>102</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>103</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>104</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>105</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>106</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>107</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>108</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>109</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>110</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>110</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="guest_choice">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>150</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Guest</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="visiteur_choice">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>200</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Visiteur</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="vendeur_choice">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendeur</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>20</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Informations personnelles</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Role</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>mdp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mdp_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>********</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
